.gitlab/deploy_6/windows.yml

```diff
@@ -34,5 +46,14 @@ deploy_staging_windows_tags-a6:
   before_script:
     - ls $OMNIBUS_PACKAGE_DIR
   script:
-    - '@("datadog-agent-6*.msi", "datadog-agent-6*.debug.zip", "datadog-agent-6*.wixpdb", "customaction-6*.pdb") |
-      ForEach-Object { $S3_CP_CMD --recursive --exclude "*" --include "$_" $OMNIBUS_PACKAGE_DIR s3://$WINDOWS_BUILDS_S3_BUCKET/tagged/ --grants read=uri=http://acs.amazonaws.com/groups/global/AllUsers full=id=3a6e02b08553fd157ae3fb918945dd1eaae5a1aa818940381ef07a430cf25732 }'
+    - $S3_CP_CMD
+      --recursive
```

Comment:
I don't think this needs to be recursive, since all the files are at the same level.
```suggestion
```